### PR TITLE
DGS 259: subsidy steps showed in a tab

### PR DESCRIPTION
**Pull request by ZijneDwaasheid**
## ID
 DGS-259

 ## Description

when clicking the 'stappen' tab we want to show the steps linked to the series

 ## Type of change

 - [ ] Bug fix
 - [x] New feature
 - [ ] Breaking change
 - [ ] Other

 ## How to test

 Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

 ## Links to other PR's

 - https://github.com/lblod/frontend-subsidie-management/pull/8

### CODE REVIEW

File: app/controllers/subsidy/measure-offer.js [hunk 1]
```diff
@@ -1,7 +1,63 @@
...
+  @action
+  async loadOptions() {
+    this.options = await this.model.series;
+    await this.latestSerie();
+  }
+
+  @action
+  updateSelection(option) {
+    this.selected = option;
+    this.selectedId = option.id;
```
Comment (wolfderechter) [inline app/controllers/subsidy/measure-offer.js]:
```suggestion
```

File: app/controllers/subsidy/measure-offer.js [hunk 2]
```diff
@@ -1,7 +1,63 @@
...
+import { action } from '@ember/object';
+import moment from 'moment';
+import { inject as service } from '@ember/service';
 
 export default class SubsidyMeasureOfferController extends Controller {
-  sort = '-modified';
-  page = 0;
-  size = 20;
+  @service router;
+  @tracked selectedId = null;
```
Comment (wolfderechter) [inline app/controllers/subsidy/measure-offer.js]:
```suggestion
```

File: app/controllers/subsidy/measure-offer.js [hunk 3]
```diff
@@ -1,7 +1,63 @@
...
+
+  @action
+  routeBack() {
+    this.router.transitionTo('subsidy.index');
+  }
+
+  @action
+  transitionToSteps() {
+    this.router.transitionTo('subsidy.measure-offer.steps', {
+      queryParams: { selectedId: this.selectedId },
```
Comment (wolfderechter) [inline app/controllers/subsidy/measure-offer.js]:
```suggestion
      queryParams: { selectedId: this.selected.id },
```
Bit redundant to create a selectedId in this case

File: app/templates/subsidy/measure-offer/steps.hbs [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+                {{#each this.model.activeApplicationFlow.sortedDefinedSteps as |step|}}
+                  <li class="au-c-list-navigation__item">
+                    {{#if step.subsidyProceduralStep.description}}
+                        <AuLink 
+                          class="au-c-list-navigation__link"
+                          style="flex-direction: column; align-items: start;"
+                          {{!template-lint-disable no-inline-styles}}
+                        >
+                        <div class="au-u-flex au-u-flex--spaced-tiny">
+                          <AuHeading @skin="4">{{step.order}}. {{step.subsidyProceduralStep.description}}</AuHeading></div>
```
Comment (wolfderechter) [inline app/templates/subsidy/measure-offer/steps.hbs]:
```suggestion
                          <AuHeading @skin="4">{{step.subsidyProceduralStep.description}}</AuHeading></div>
```
The step order is not needed here after all, maybe in a future feature where we can visualize the active, and 'skipped' steps

File: app/templates/subsidy/measure-offer.hbs [hunk 5]
```diff
@@ -97,4 +182,4 @@
       </AuDataTable>
      </div>
   </div>
-</main>
+</main> --}}
```
Comment (wolfderechter) [inline app/templates/subsidy/measure-offer.hbs]: should we cleanup this commented code?

**Review by wolfderechter — APPROVED** [review]
good to go, 2 small remarks

File: app/templates/subsidy/measure-offer.hbs (hunk 5 above)
Comment (ZijneDwaasheid) [reply app/templates/subsidy/measure-offer.hbs]: ah yes indeed this can go

**Pull request merged**